.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: 💡 Feature request
+about: Suggest an enhancement to the Layer5 website
+title: ''
+labels: 'kind/enhancement'
+assignees: ''
+---
+**Current Behavior**
+<!-- A brief description of what the problem is. (e.g. I need to be able to...) -->
+
+**Desired Behavior**
+<!-- A brief description of the enhancement. -->
+
+---
+**Screenshots / Mockups**
+<!-- Add any other context or screenshots about the feature request here. -->
+
+**Alternatives**
+<!-- A brief description of any alternative solutions or features you've considered. -->
+
+---
+**Contributor Resources**
```

Comment:
@amino19 Refer this, https://github.com/layer5io/gowrk2/pull/15/files#r663357864